Error with latest Godot dev build

50%] Compiling ==> modules\voxel\streams\voxel_block_serializer.cpp
voxel_block_serializer.cpp
modules\voxel\streams\voxel_block_serializer.cpp(90): error C3861: 'ERR_FAIL_COND_V_MSG': identifier not found
modules\voxel\streams\voxel_block_serializer.cpp(119): error C3861: 'ERR_FAIL_COND_V_MSG': identifier not found
modules\voxel\streams\voxel_block_serializer.cpp(164): error C3861: 'ERR_FAIL_COND_V_MSG': identifier not found
modules\voxel\streams\voxel_block_serializer.cpp(167): error C3861: 'ERR_FAIL_COND_V_MSG': identifier not found
scons: *** [modules\voxel\streams\voxel_block_serializer.windows.debug.64.obj] Error 2
scons: building terminated because of errors.

Using the latest commit 35444576ff and VT master, I can confirm it won't build. but I get different error messages.

```
Compiling ==> modules\voxel\streams\voxel_stream_image.cpp
modules\voxel\streams\voxel_stream.cpp(22): error C3861: 'ERR_EXPLAIN': identifier not found
Compiling ==> modules\voxel\streams\voxel_stream_noise.cpp
modules\voxel\streams\voxel_stream.cpp(43): error C3861: 'ERR_EXPLAIN': identifier not found
voxel_stream_noise.cpp
```

This PR changed the error macros. 
https://github.com/godotengine/godot/pull/33391

Working on a PR.

--- a/streams/voxel_stream.cpp
+++ b/streams/voxel_stream.cpp
@@ -18,13 +18,11 @@ void VoxelStream::emerge_block(Ref<VoxelBuffer> out_buffer, Vector3i origin_in_v
 		const Variant *args[3] = { &arg1, &arg2, &arg3 };
 		Variant::CallError err;
 		script->call("emerge_block", args, 3, err);
-		if (err.error != Variant::CallError::CALL_OK) {
-			ERR_EXPLAIN("voxel_stream.cpp:emerge_block gave an error: " + String::num(err.error) +
-						", Argument: " + String::num(err.argument) +
-						", Expected type: " + Variant::get_type_name(err.expected));
-			ERR_FAIL();
-			// This had to be explicitely logged due to the usual GD debugger not working with threads
-		}
+		ERR_FAIL_COND_MSG(err.error != Variant::CallError::CALL_OK,
+			"voxel_stream.cpp:emerge_block gave an error: " + String::num(err.error) +
+			", Argument: " + String::num(err.argument) +
+			", Expected type: " + Variant::get_type_name(err.expected));
+		// This had to be explicitely logged due to the usual GD debugger not working with threads
 	}
 }
 
@@ -39,13 +37,11 @@ void VoxelStream::immerge_block(Ref<VoxelBuffer> buffer, Vector3i origin_in_voxe
 		const Variant *args[3] = { &arg1, &arg2, &arg3 };
 		Variant::CallError err;
 		script->call("immerge_block", args, 3, err);
-		if (err.error != Variant::CallError::CALL_OK) {
-			ERR_EXPLAIN("voxel_stream.cpp:immerge_block gave an error: " + String::num(err.error) +
-						" Argument: " + String::num(err.argument) +
-						" Expected type: " + Variant::get_type_name(err.expected));
-			ERR_FAIL();
-			// This had to be explicitely logged due to the usual GD debugger not working with threads
-		}
+		ERR_FAIL_COND_MSG(err.error != Variant::CallError::CALL_OK,
+			"voxel_stream.cpp:immerge_block gave an error: " + String::num(err.error) +
+			" Argument: " + String::num(err.argument) +
+			" Expected type: " + Variant::get_type_name(err.expected));
+		// This had to be explicitely logged due to the usual GD debugger not working with threads
 	}
 }
 

